Alignment/OfflineValidation/test/eopTreeWriter_cfg.py

```diff
@@ -23,7 +23,7 @@
 
 # load configuration files
 process.load("Configuration.StandardSequences.MagneticField_cff")
-process.load("Configuration.Geometry.GeometryDB_cff")
+process.load("Configuration.StandardSequences.Geometry_cff")
```

Comment:
This change of include will give you a wrong geometry. I think, you need to use:
```
process.load("Configuration.Geometry.GeometryRecoDB_cff")
```
BTW, this comment is valid for all other similar changes.